### ISSUE TITLE
Add body text to email notifier resolved notifications

### ISSUE THREAD

**Issue by axeal** [rancher/rancher#24156]
**What kind of request is this (question/bug/enhancement/feature request):**

Enhancement

**Steps to reproduce (least amount of steps as possible):**

In `/etc/alertmanager/config/notification.tmpl` the email notifier body generation contains the following logic:

```
{{- define "__email_text_list" -}}
{{ range .Alerts.Firing }}
{{ template "__email_text_single" . }}
{{ end -}}
{{ end -}}
```

As a result firing notifications will contain a body such as the following:

```
Alert Name: Pod Deployment error
Severity: warning
Cluster Name: custom (ID: c-b9wxr)
Workload Name: nginx
Target: default:nginx-6bf66d498b-dfmcl
Count: 1
Event Message: Failed to pull image "axeal/foo:bob": rpc error: code = Unknown desc = Error response from daemon: pull access denied for axeal/foo, repository does not exist or may require 'docker login'
First Seen: 2019-11-19 10:30:24 +0000 UTC
Last Seen: 2019-11-19 10:30:24 +0000 UTC
```

However, the resolved notification will not contain any body providing details of the cluster and deployment the error related to, but just the subject, e.g. `[Resolved]Warning event of Pod occurred`. As a result with multiple alerts firing it is not possible to determine from the Resolved notification which alert is resolved.

Enhancement request is therefore to add body text to resolved email notifications for `Alerts.Resolved` to show details of the resolved alert(s).

**Environment information**
- Rancher v2.3.2

**Comment by deniseschannon** [comment]
Per the PR:

Before:
![image](https://user-images.githubusercontent.com/21168270/78616390-ad8fad00-78a6-11ea-9075-5580aaa1949d.png)


After:

![image](https://user-images.githubusercontent.com/21168270/78616429-c4360400-78a6-11ea-81f9-afd13f589f15.png)

**Comment by jiaqiluo** [comment]
The bug is reproduced in v2.4.2 

the "resolved" email has no content 

<img width="637" alt="Screen Shot 2020-04-09 at 3 49 59 PM" src="https://user-images.githubusercontent.com/6218999/78947313-d87e2a80-7a79-11ea-8bd8-77d9a55e9fb0.png">

**Comment by jiaqiluo** [comment]
The bug fix is validated on v2.4-head 8439d8bde  and master-head c4e175cf6

<img width="521" alt="Screen Shot 2020-04-10 at 9 27 07 AM" src="https://user-images.githubusercontent.com/6218999/79007563-407f5000-7b10-11ea-8d5b-f8b74e3c959e.png">
